docs/src/App/App.js

```diff
@@ -25,23 +46,62 @@ export default class App extends Component {
   }
 
   render() {
+    const { menuOpen } = this.state;
+
     return (
       <ThemeProvider theme={themes.wireframe}>
-        <Box
-          paddingTop="large"
-          paddingBottom="xsmall"
-          paddingLeft="gutter"
-          paddingRight="gutter"
-        >
-          <Link to="/" style={{ display: 'inline-block' }}>
-            <Logo />
-          </Link>
-        </Box>
+        <div className={styles.header}>
```

Comment:
Good question. I'm trying to get out of the habit of applying classes to components, so that there's zero risk of style clashes (i.e. imagine `className` was renamed to `dangerouslySetClassName`).